apis/projectcontour/v1/httpproxy.go

```diff
@@ -1168,3 +1171,34 @@ type HTTPProxyList struct {
 	metav1.ListMeta `json:"metadata"`
 	Items           []HTTPProxy `json:"items"`
 }
+
+// SlowStart will gradually increase amount of traffic to a newly added endpoint.
+// It works only with RoundRobin and WeightedLeastRequest load balancing strategies.
```

Comment:
Maybe worth explicitly stating that it will be _ignored_ if using a different load balancing strategy.

Reply:
I have added a note that the configuration is ignored for other strategies.  But on second thought, I started to question myself, so I want to ask: should we ignore (like it is done now) or make HTTPProxy invalid?